manifests/ca/service.pp

```diff
@@ -1,7 +1,19 @@
+# **NOTE: THIS IS A [PRIVATE](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private) DEFINED TYPE**
+#
+# Start a CA service on the given port and wait until the service has
+# successfully started or the timeout is reached.
+#
+# @param port
+#   The port upon which to listen
+#
+# @param timeout
```

Comment:
@lnemsick-simp Not really. The timeout is the timeout but there is a 2 second sleep between tries. For some reason this is all that would work. So, it's more like timeout+2 but I felt like that was too down in the weeds for the users and would cause confusion.